### DISCUSSION TITLE
Hello.my GitHub account does not have verified academic email address!

### DISCUSSION THREAD

**Discussion by sh776** [RosettaCommons/rosetta#135] [ROSIE]
Hello. My GitHub account does not have a verified academic email address! 
https://en.pasteur.ac.ir
Email : info@pasteur.ac.ir.

**Comment by roccomoretti** [comment]
Unfortunately, there are legal issues with using servers in the United States to provide computational services to people in Iran, which means that we are unable to approve Iranian email addresses for ROSIE accounts at this time.

We apologize for the current state of the world.
